src/formatly.ts

```diff
@@ -56,7 +56,7 @@ export async function formatly(
 		ran: true,
 		result: await new Promise((resolve, reject) => {
 			const child = spawn(baseCommand, [...args, ...patterns], {
-				cwd: process.cwd(),
+				cwd: options.cwd ?? process.cwd(),
```

Comment:
[Refactor] `resolveFormatter` also has cwd defaulting logic:

https://github.com/JoshuaKGoldberg/formatly/blob/7b8cddbadc78b9fb1c6c0b353bac6679f95c1b98/src/resolveFormatter.ts#L6-L7

...which I think comes out to basically the same thing as `?? process.cwd()`. WDYT about, say, having a single cwd variable set in this function that's passed down later? As in, only having one place where cwd has a default set?

Reply:
Done, but this means the only argument to `resolveFormatter` is no longer optional.

Reply:
Oh, hmm, you're right. Darn. Having `resolveFormatter`'s argument be optional is really nice for end-users of it... I guess we'll just have to live with the duplicate defaulting. I'll switch it back - sorry for the back-and-forht.